docs/resources/connection.md

```diff
@@ -488,6 +487,6 @@ Attributes exported by this resource include:
 
 Connections can be imported using their id, e.g.
 
-```
-$ terraform import auth0_connection.google con_a17f21fdb24d48a0
+```terminal
```

Comment:
I don't believe `terminal` is a supported option. From my brief research, it looks like `shell` might be more appropriate.